app/api/chemotion/profile_api.rb

```diff
@@ -91,8 +93,14 @@ class ProfileAPI < Grape::API
         declared_params = declared(params, include_missing: false)
         data = current_user.profile.data || {}
         available_ements = API::ELEMENTS + Labimotion::ElementKlass.where(is_active: true).pluck(:name)
-
-        data['layout'] = { 'sample' => 1, 'reaction' => 2, 'wellplate' => 3, 'screen' => 4, 'research_plan' => 5 } if data['layout'].nil?
+        data['layout'] = {
+          'sample' => 1,
+          'reaction' => 2,
+          'wellplate' => 3,
+          'screen' => 4,
+          'research_plan' => 5,
+          'cell_line' => -1000,
```

Comment:
Should the arbitrary use of -1000 be changed to something more appropriate?

Reply:
i chose that number to avoid conflict if a user has many invisible generic elements. It is most unlikely that there are so many to be conflicted with -1000